src/pages/ProfilesExplorerView/components/SceneProfilesExplorer/SceneProfilesExplorer.tsx

```diff
@@ -441,12 +458,17 @@ export class SceneProfilesExplorer extends SceneObjectBase<SceneProfilesExplorer
                 <refreshPickerControl.Component key={refreshPickerControl.state.key} model={refreshPickerControl} />
               )}
 
-              <IconButton
-                name="share-alt"
-                tooltip="Copy shareable link to the clipboard"
-                onClick={actions.onClickShareLink}
-              />
-              <VersionInfoTooltip />
+              <div className={styles.miscButtons}>
+                <IconButton name="cog" tooltip="View/edit user settings" onClick={actions.onClickUserSettings} />
```

Comment:
**suggestion (non-blocking)**: Add a back/close button to Settings View

WIth this change the Settings View is going to be more integrated with the page and it feel it would be good to have close/back button on it. Once user opens it there's no clear way to go back  to main view.

Reply:
I believe they will click on the browser's back button, which will lead them where they were.

Reply:
Just in case, added in https://github.com/grafana/explore-profiles/pull/172/commits/ed1926b923c1bc9c06341afeaefc58a45c99c9fe